Dockerfile

```diff
@@ -8,4 +8,17 @@ RUN \
   rm -rf /var/lib/apt/lists/* && \
   rm -rf /var/cache/oracle-jdk8-installer
 
+RUN \
+  apt-get update && \
+  apt-get install -y libnss3 libfontconfig
+
+RUN \
+  apt-get update && \
+  apt-get install -y libxss1 libappindicator1 libindicator7 libasound2 fonts-liberation xdg-utils && \
+  wget -N https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb && \
+  sudo dpkg -i google-chrome-stable_current_amd64.deb && \
+  sudo apt-get -f install -y && \
+  sudo dpkg -i google-chrome-stable_current_amd64.deb
```

Comment:
Cannot find the material i got it from, but the url you sent to me does the same thing here https://gist.github.com/ziadoz/3e8ab7e944d02fe872c3454d17af31a5

Thats because if the first `dpkg` face any issue (e.g. xxx.xxx is not installed, etc.) then the repeated`dpkg` will make it work. XD